src/main/java/seedu/exercise/model/exercise/Exercise.java

```diff
@@ -1,16 +1,16 @@
-package seedu.address.model.exercise;
+package seedu.exercise.model.exercise;
 
-import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.exercise.commons.util.CollectionUtil.requireAllNonNull;
 
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
 
-import seedu.address.model.tag.Muscle;
+import seedu.exercise.model.tag.Muscle;
 
 /**
- * Represents a Person in the quantity book.
+ * Represents an Exercise in the quantity book.
```

Comment:
Need to change "quantity book" to "exercise book".

Reply:
done